client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-how-to-migrate/index.tsx

```diff
@@ -76,29 +77,41 @@ const SiteMigrationHowToMigrate: FC< Props > = ( props ) => {
 		? true
 		: false;
 
-	const { updateMigrationStatus } = useUpdateMigrationStatus();
+	const [ isLoading, setIsLoading ] = useState( false );
+	const { updateMigrationStatusAsync } = useUpdateMigrationStatus();
 
-	const handleClick = ( how: string ) => {
+	const handleClick = async ( how: string ) => {
 		const destination = canInstallPlugins ? 'migrate' : 'upgrade';
 		if ( site?.ID ) {
 			const parsedHow = how === HOW_TO_MIGRATE_OPTIONS.DO_IT_MYSELF ? 'diy' : how;
-			updateMigrationStatus( site.ID, `migration-pending-${ parsedHow }` );
+			try {
+				setIsLoading( true );
+				await updateMigrationStatusAsync( site.ID, `migration-pending-${ parsedHow }` );
+			} catch ( error ) {
+				recordTracksEvent( 'calypso_site_migration_how_to_update_status_request_error' );
+			} finally {
+				setIsLoading( false );
+			}
 		}
 		return navigation.submit?.( { how, destination } );
 	};
 
 	const stepContent = (
 		<>
-			<div className="how-to-migrate__list">
-				{ options.map( ( option, i ) => (
-					<FlowCard
-						key={ i }
-						title={ option.label }
-						text={ option.description }
-						onClick={ () => handleClick( option.value ) }
-					/>
-				) ) }
-			</div>
+			{ isLoading ? (
+				<LoadingEllipsis />
```

Comment:
It would be nice to centralize the loader. Since the step content is centered, it looks a little weird having the loader in the left.

Reply:
I swear that was centered while I was testing! 😅 It's [fixed](https://github.com/Automattic/wp-calypso/pull/95612/files#diff-263e4afa19ba8a95b4c2662c5dfc0a0b97103d852c7b20c18c957a382dbdf8c6R47) now.